packages/app/src/cli/services/dev/migrate-app-module.test.ts

```diff
@@ -49,62 +73,58 @@ describe('getPaymentsExtensionsToMigrate()', () => {
     const remoteExtension = getRemoteExtension({
       type: 'payments_app_credit_card',
       title: 'Ten Chars Ten Chars Ten Chars Ten Chars Ten 123456789',
-      uuid: 'yy',
     })
 
     // When
-    const toMigrate = getPaymentsExtensionsToMigrate([localExtension], [remoteExtension], defaultIds)
+    const toMigrate = getModulesToMigrate([localExtension], [remoteExtension], {}, defaultMap)
 
     // Then
     expect(toMigrate).toStrictEqual([{local: localExtension, remote: remoteExtension}])
   })
 
-  test('matching my local and remote IDs', () => {
+  test('matching different title and localIdentifier, using uuid', () => {
     // Given
-    const localExtension = getLocalExtension({type: 'payments_extension', localIdentifier: 'my-action'})
-    const remoteExtension = getRemoteExtension({type: 'payments_app_credit_card', title: 'remote', uuid: '1234'})
+    const localExtension = getLocalExtension({type: 'payments_extension', localIdentifier: 'module-A'})
+    const remoteExtension = getRemoteExtension({type: 'payments_app_credit_card', title: 'random-title', uuid: '1234'})
```

Comment:
To make it a bit more clear that we're matching remoteExtensions by the `defaultIdentifiers` that's being passed in to `getModulesToMigrate`, what do you think about changing how we're setting the `uuid` value here? 

```suggestion
    const remoteExtension = getRemoteExtension({type: 'payments_app_credit_card', title: 'random-title', uuid: defaultIdentifiers['moduleA']})
```